webui/react/src/components/OverviewStats.module.scss

```diff
@@ -3,7 +3,7 @@
   flex-direction: column;
   justify-content: space-between;
 
-  .title {
+  .title * {
```

Comment:
maybe instead of overwriting the title CSS this should just be a Label?

Reply:
✅